TypeSystem/flow-analysis/reachability_while_A03_t01.dart

```diff
@@ -8,11 +8,11 @@
 /// - Let `before(S) = split(true(E))`.
 /// - Let `after(N) = inheritTested(join(false(E), unsplit(break(S))), after(S))`
 ///
-/// @description Checks that if `E` is not a boolean literal flow analysis
-/// detects assignment in `S`.
+/// @description Checks that if `E` is a boolean `true` literal and there is no
+/// reachable `break` or `return` in `S` then `after(N) is unreachable.
```

Comment:
I don't think a `return` statement in `S` could make `after(N)` reachable. How would that happen?